crates/ditto-codegen-js/src/convert.rs

```diff
@@ -232,6 +232,31 @@ fn convert_expression(
                 })
                 .collect(),
         },
+
+        // The cheap/lazy way to do it
```

Comment:
I think at this stage I'd prefer to generate a [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) here.

In future, I'd want to generate an `if ... else` block like this only if we're inside an `Effect` expression (see #9), so this will come in useful then 👍